lib/values/string_constants.dart

```diff
@@ -0,0 +1,27 @@
+class AppStrings {
+  static const welcome = 'Welcome';
```

Comment:
Please specify data type for public APIs.

Reply:
Resolved